src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -369,7 +379,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     ============================================================
   -->
   <Target Name="_GetRestoreProjectStyle"
-    DependsOnTargets="_GetProjectJsonPath"
+    DependsOnTargets="_GetProjectJsonPath;CollectPackageReferences"
```

Comment:
Why does this need a target to answer this question? Targets are really, really damn expensive.